features/step_definitions/metric_configuration_steps.rb

```diff
@@ -18,6 +18,18 @@
                                               kalibro_configuration_id: @kalibro_configuration.id})
 end
 
+Given(/^I have a hotspot metric configuration within the given kalibro configuration$/) do
+  @hotspot_metric_configuration = FactoryGirl.create(:metric_configuration,
+                                                     {metric: FactoryGirl.build(:hotspot_metric),
+                                                      reading_group_id: @reading_group.id,
```

Comment:
I think hotspot metric configurations do not require reading group associations.